include/zephyr/linker/linker-tool-mwdt.h

```diff
@@ -90,9 +89,10 @@
  */
 #ifdef CONFIG_XIP
 #define SECTION_DATA_PROLOGUE(name, options, align) \
-	name options ALIGN(8) align :
+	name options ALIGN(8) : align
```

Comment:
So this new syntax is suitable for both ARC and RISC-V isa targets?

Reply:
yes

Reply:
Ok, can confirm - it works even for old MWDT versions.